staging/src/k8s.io/client-go/dynamic/simple.go

```diff
@@ -293,7 +293,20 @@ func (c *dynamicResourceClient) Get(ctx context.Context, name string, opts metav
 	if err != nil {
 		return nil, err
 	}
-	return uncastObj.(*unstructured.Unstructured), nil
+
```

Comment:
rather than trying to compensate for an unexpected result, would it be better to tell the decode method what we want it to decode into?

```go
obj := &unstructured.Unstructured{}
runtime.DecodeInto(unstructured.UnstructuredJSONScheme, retBytes, obj)
```

Reply:
updated -- yes this is much better.  thanks